.vscode/settings.json

```diff
@@ -17,6 +17,7 @@
     "editor.tabSize": 2,
     "cSpell.words": [
```

Comment:
can you display what is the benefit of this : 
```
"cSpell.words": [
 "backgroundposition",
        "isplayer",
        "بلنتي"
    ]
    ```

Reply:
it is for spell checker